config/api.yml

```diff
@@ -2315,6 +2316,53 @@
       :post:
       - :name: remove
         :identifier: svc_catalog_provision
+  :cloud_templates:
+    :description: Cloud Templates
+    :identifier: template
+    :options:
+      - :collection
+      - :subcollection
+    :verbs: *gpd
+    :klass: ManageIQ::Providers::CloudManager::Template
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: miq_template_show_list
+      :post:
+      - :name: query
+        :identifier: miq_template_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: miq_template_show
+      :post:
+      - :name: delete
+        :identifier: image_delete
+      - :name: query
+        :identifier: miq_template_show
+      - :name: create
```

Comment:
create would not be a resource action but a collection action. Can you also add tests for this?

Reply:
I'll just remove `create` and `delete` actions for now, and add it, when it's ready. Does it make sense?